packages/google_sign_in/google_sign_in/ios/Classes/FLTGoogleSignInPlugin.m

```diff
@@ -240,9 +241,11 @@ - (void)signIn:(GIDSignIn *)signIn
 #pragma mark - private methods
 
 - (void)respondWithAccount:(id)account error:(NSError *)error {
-  FlutterResult result = _accountRequest;
-  _accountRequest = nil;
-  result(error != nil ? getFlutterError(error) : account);
+  dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Can you explain why you added this? It doesn't look like the docs say this comes back on a different thread. Did you see this happening in production?

Reply:
That was more a stab in the dark at addressing someone else's crash (which I couldn't reproduce myself): https://github.com/flutter/flutter/issues/44564#issuecomment-765896967 , and I had intended to revert it after learning that it didn't fix his problem.

I've since moved off my macincloud subscription where I created this PR, so when I get a chance I'll need to set this up in my new dev environment.